server/integration/endpoint/src/main/resources/schema/jboss-infinispan-endpoint_8_0.xsd

```diff
@@ -18,6 +18,11 @@
                 <xs:documentation>The socket on which this connector will bind itself</xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="ignored-caches" type="tns:ignored-caches">
```

Comment:
In theory this should have justified bumping the schema to 8.1... but I'm tempted to let you off just for this time, since the change is additive and quite minor